java/src/test/java/org/apache/arrow/gandiva/evaluator/ProjectorTest.java

```diff
@@ -94,10 +94,19 @@ public void testMakeProjector() throws GandivaException {
     ExpressionTree expr = TreeBuilder.makeExpression(ifNode, Field.nullable("c", int64));
     List<ExpressionTree> exprs = Lists.newArrayList(expr);
 
+    long startTime = System.currentTimeMillis();
```

Comment:
pls add unit tests in cpp - if everything is same, module should be same. if anything changes, module should be different